anomdtct/entrypoint

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+"""
+Entrypoint script for running pipeline.replace_single_day.
+"""
+
+import argparse
+
+from anomdtct.pipeline import replace_single_day
+from google.cloud import bigquery
+from google.cloud.bigquery_storage_v1beta1 import BigQueryStorageClient
+
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument("dt", help="model date to process")
+parser.add_argument("--project-id", "--project_id", help="destination project")
+parser.add_argument("--dataset-id", "--dataset_id", help="destination dataset")
+parser.add_argument("--table-id", "--table_id", help="destination table")
+parser.add_argument("--spreadsheet-id", "--table_id", help="ID of the Google spreadsheet")
+parser.add_argument("--spreadsheet-key", "--table_id", help="Key for authenticating to Google spreadsheet")
```

Comment:
This key should be set as configuration variable in Airflow

Reply:
Do we know if command-line arguments show up in logs in Airflow?

Reply:
Even better, we should avoid sending a key at all. I assume we can set permissions on the spreadsheet to allow edits from the Airflow service account, so we can use the same creds as the container is using to access BQ.